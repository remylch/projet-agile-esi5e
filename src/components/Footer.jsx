import React from "react";

/**
 * @description Footer component
 */
function Footer() {
  return (
    <div
      data-testid="footer"
      className="h-14 bg-white shadow-sm flex items-center justify-center"
    >
      <h6 data-testid="title-footer">SYNTAX MAP</h6>
    </div>
  );
}

export default Footer;
