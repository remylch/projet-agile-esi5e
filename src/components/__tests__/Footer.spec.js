import Footer from "../Footer";
import { render, screen, cleanup } from "@testing-library/react";

afterEach(() => {
  cleanup();
});

test("Render the footer", () => {
  render(<Footer />);
  const el = screen.getByTestId("footer");
  expect(el).toMatchSnapshot("<h6>");
});

test("Render the text in the footer", () => {
  render(<Footer />);
  const el = screen.getByTestId("title-footer");
  expect(el).toHaveTextContent("SYNTAX MAP");
});
