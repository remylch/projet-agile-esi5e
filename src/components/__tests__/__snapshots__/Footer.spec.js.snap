// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render the footer: <h6> 1`] = `
<div
  class="h-14 bg-white shadow-sm flex items-center justify-center"
  data-testid="footer"
>
  <h6
    data-testid="title-footer"
  >
    SYNTAX MAP
  </h6>
</div>
`;
